paparazzi-gradle-plugin/src/test/projects/missing-supported-plugins/build.gradle

```diff
@@ -0,0 +1,3 @@
+plugins {
+  id 'app.cash.paparazzi'
+}
```

Comment:
we already have a test for something similar, i believe.  is this a renaming of that?  if not, can we reuse that?

Reply:
`missing-library-plugin`

Reply:
the difference between them is the feature toggle. I can reuse `missing-library-plugin` project, but I have to manually turn on feature toggle in the test, is it fine?

Reply:
Oops, completely forgot about that for some reason lol.  Nah, what you have makes sense to keep test coverage on both sides.  Please ignore my earlier comment!